igor-web/config/igor.yml

```diff
@@ -10,6 +10,13 @@ jenkins:
       username: 'username'
       password: 'password'
 
+dockerRegistry:
```

Comment:
Please prepare a companion PR for spinnaker/spinnaker that updates the 'unified' config to reflect this new config: https://github.com/spinnaker/spinnaker/blob/master/config/igor.yml